monitor/src/main/webapp/resources/monitor/reload.js

```diff
@@ -0,0 +1,21 @@
+window.setTimeout(function() {
+	$.ajax({
+		type : "GET",
+		dataType : "json",
+		url : "/monitor/admin/apps/reload",
+		success : function(data) {
+			data.forEach(update)
+		},
+		error : function(xhr, status, error) {
+			var err = eval("(" + xhr.responseText + ")");
```

Comment:
is the response text not in `error`? I would always be careful using `eval` since it opens your app up for attacks very easily.